### PR TITLE
Lint all CSS files

### PR DESCRIPTION
**Pull request by mikachan**
As a way to bring more of the codebase into the correct format, this PR lints all of the CSS files. This is the result of running `npm run lint:css:fix`.

### CODE REVIEW

File: src/demo-text-input/demo-text-input.css [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
 
 .demo-text-input .components-range-control__root {
-    flex-direction: row-reverse;
+	flex-direction: row-reverse;
 }
 
 .demo-text-input .components-range-control__number {
-    margin-left: unset !important;
-    margin-right: 1rem;
-}
```
Comment (madhusudhand) [inline src/demo-text-input/demo-text-input.css:27]: It seem to conflict with PHP lint rule where new line at the end of file is required.

Comment (mikachan) [reply src/demo-text-input/demo-text-input.css:27]: There's a new line at the end of each of the CSS files. When you run the linter, does it remove or not add new lines at the end?

Comment (madhusudhand) [reply src/demo-text-input/demo-text-input.css:27]: Sorry my bad. diff caused the confusion. 
I can confirm that it ads additional line at the end of the file.

**Review by madhusudhand — APPROVED** [review]
LGTM!

File: src/demo-text-input/demo-text-input.css (hunk 1 above)
Comment (mikachan) [reply src/demo-text-input/demo-text-input.css:27]: No worries! I wasn't sure if my editor settings were adding the new line or if it was the settings from the repo. My editor settings kept tripping me up before with this 😅

**Pull request merged**